docs/changelog.txt

```diff
@@ -63,6 +63,8 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 - Window behavior: non-resizable windows now allow dragging by their frame edges by default
 - `gui/autodump`: fort-mode keybinding: Ctrl-H
 - Window behavior: if you have multiple DFHack tool windows open, scrolling the mouse wheel while over an unfocused window will focus it and raise it to the top
+- ``Textures.load_textures()``: can now load spritesheets with tilesizes other than 8x12.
+- ``gui/pathable``: now uses 32x32 sprites.
```

Comment:
The `Textures::load_textures()` line should go under API, and the `gui/pathable` line won't be user-visible as a change in behavior so doesn't need to be mentioned.

Also, there has been a release, so the changelog changes will need to be synced and moved up to the new "Future" section.

Reply:
i'm always glad when i _don't_ have to write changelog entries! 😂 
but yeah, i'll move that entry in the right section then